packages/communication-react/review/communication-react.api.md

```diff
@@ -759,7 +759,7 @@ export type ChatCompositeOptions = {
     errorBar?: boolean;
     participantPane?: boolean;
     topic?: boolean;
-    autoFocus?: 'sendBoxTextField' | false;
+    autoFocus?: 'sendBoxTextField' | 'disabled';
```

Comment:
Is there an advantage to having `'disabled'` instead of just `undefined`?

Reply:
Do we autofocus by default if nothing is passed in here?

Reply:
the only thing I can think of is that it will just provide clarity of what autofocus would do. having it undefined might hint it would go do a 'default' behavior instead of explicitly say there will be no autofocus present

Reply:
we can remove it, the default is no focus

Reply:
Adding 'disabled' now deviates from our composite config options.
As a practice, we decided if an optional prop is undefined, it means that that feature is disabled.
What do you think @PorterNan @prprabhu-ms ?